mypy.ini

```diff
@@ -21,3 +21,6 @@ warn_redundant_casts = True
 warn_return_any = True
 no_implicit_reexport = True
 strict_equality = True
+
+[mypy-importlib_metadata.*]
+ignore_missing_imports = True
```

Comment:
This is now necessary since `importlib_metadata` was only flagged to be typed from `3.4.0` onward. LMK if you want this version as minimum instead.